src/ExtInterface/ReefMod/Domain.jl

```diff
@@ -309,12 +309,27 @@ function load_initial_cover(::Type{ReefModDomain}, data_path::String, loc_ids::V
         icc_data[:, :, i] = Matrix(CSV.read(fn, DataFrame; drop=[1], header=false, comment="#"))
     end
 
+    # use ReefMod distribution for coral size class population (shape parameters have units log(cm^2))
+    # as suggested by YM (pers comm. 2023-08-08 12:55pm AEDT)
+    reef_mod_area_dist = LogNormal(log(700), log(4))
+    # get bin edges for coral diameters and transform to coral areas in cm^2
+    bin_edges_area = colony_mean_area(Float64[0, 2, 5, 10, 20, 40, 80])
```

Comment:
Please use sentence case for comments that are not inline.

Best practice is to comment not what is being done but _why_.
The occasions you include the what is when the code is necessarily complicated.

The comment for the code here for example is unnecessary. You should write more readable code (use informative variable names) instead.

```julia
# get bin edges for coral diameters and transform to coral areas in cm^2
bin_edges_area = colony_mean_area(Float64[0, 2, 5, 10, 20, 40, 80])
```